.travis.yml

```diff
@@ -16,4 +16,10 @@ jobs:
       name: "Unit Tests"
       script: npm run test:lint && npm run test:unit && npm run test:danger && npm run test:git-history
     - name: "Integration Tests"
-      script: ./scripts/run-integration-tests.travis.sh
\ No newline at end of file
+      script: ./scripts/run-integration-tests.travis.sh
+    - stage: release
+      if: branch = master
+      script:
+       - export REPO=oneamex/one-app
+       - docker build -f Dockerfile -t $REPO .
```

Comment:
we need to build and publish both the dev image (that one app runner and such tooling will use for local dev and tests) as well as the production image.

https://github.com/americanexpress/one-app/blob/master/Dockerfile#L31 is what we use to build the dev image.